unittests/TreeStream/TreeStreamTest.cpp

```diff
@@ -19,7 +19,7 @@ TEST(TreeStreamTest, Basic) {
 
   std::vector<unsigned char> out;
   tsw.readStream(tos.getID(), out);
-  ASSERT_EQ(out.size(), 7);
+  ASSERT_EQ(7u, out.size());
   
   for (unsigned char c = 'a'; c <= 'g'; c++)
     ASSERT_EQ(out[c - 'a'], c);
```

Comment:
@andreamattavelli If you are already here, maybe you can change this order as well